### PR TITLE
UIF-397 Implement `Rollover logs` view

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose

https://issues.folio.org/browse/UIF-397
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/Ledger/RolloverLogs/constants.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { omit } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  OVERALL_ROLLOVER_STATUS,
+  LEDGER_ROLLOVER_TYPES,
+} from '../../common/const';
+
+export const LOGS_RESULTS_PANE_TITLE = <FormattedMessage id="ui-finance.actions.rolloverLogs" />;
```
Comment (AndreiShumski2) [inline src/Ledger/RolloverLogs/constants.js:9]: Do you really need this const?

**Review by AndreiShumski2 — APPROVED** [review]

File: src/Ledger/RolloverLogs/constants.js (hunk 1 above)
Comment (usavkov-epam) [reply src/Ledger/RolloverLogs/constants.js]: actually no, thx)

**Review by zburke — COMMENTED** [review]
@usavkov-epam, adding a new required interface is a breaking change. Either revert this PR, make the new interfaces optional, or issue a new PR that bumps the version from `3.2.0` to `4.0.0`.

**Comment by usavkov-epam** [comment]
@zburke Thank you for your comment! Currently we remove new interfaces in #666 until the backend implements them. We will do as you wrote above in a new PR when we add the interfaces back.

**Pull request merged**